challenges-14-plus.yml

```
@@ -0,0 +1,472 @@
+  
+          ```yml
+          'data-turbo-track': reload
+          ```
+
+          To both `link_attributes` and `script_attributes`.
+        b: |
+          By making sure you enable versioning in Webpack Encore:
+          
+          ```javascript
+          .enableVersioning(Encore.isProduction())
+          ```
+          
+          And by modifying the file `config/packages/webpack_encore.yaml` and add the attribute:
+          
+          ```yml
+          'data-turbo-track': reload
+          ```
+          
+          To both `link_attributes` and `script_attributes`.
+        c: |
+          By modifying the file `config/packages/webpack_encore.yaml` and add the attribute:
+  
+          ```yml
+          'data-turbo-track': refresh
+          ```
+
+          To `script_attributes`.
+        d: |
+          By making sure you enable versioning in Webpack Encore:
+  
+          ```javascript
+          .enableVersioning(Encore.isProduction())
+          ```
+
+          And by modifying the file `config/packages/webpack_encore.yaml` and add the attribute:
+
+          ```yml
+          'data-turbo-webpack': track-versioning
+          ```
+
+          To both `link_attributes` and `script_attributes`.
+      correct: b
+
+manual_visit:
+  question: |
+    In our website there is a section that is very special. Only a certain type
+    of users can enter, so I've decided to pop up a big dialog in the front page
+    of that section so that users can *leave* if they don't feel up to it!
+    
+    Here's how it looks!
+    
+    ```html
+    <div id="my_great_modal" style="display: none">
+        <h1>Welcome to our Pizza Section!</h1>
+
+        <p>You need to be a pizza lover before you proceed!</p>
+
+        <button
+            class="btn btn-primary"
+            onclick="javascript:document.getElementById('my_great_modal').style.display='none';"
+        >
+            Proceed
+        </button>
+    </div>
+    ```
+    
+    This works as expected, but I'm a bit nervous about leaving too much time
+    on our users to decide. They should be quick! So I've added a timeout on
+    the main JavaScript of the page to force users *away* after a certain
+    amount of time.
+    
+    ```javascript
+    window.setTimeout(() => {
+        window.location.href = '/';
+    }, 10000);
+    ```
+    
+    How can we use Turbo Drive (instead of just `window.location.href=`)
+    to perform a page visit without doing a full refresh?
+  explanation: |
+    By using the syntax `import * as Turbo from '@hotwired/turbo'` you will be importing all
+    public functions from the library into an object called `Turbo`. This enables
+    you to use some Turbo Drive specific actions in your scripts, like `Turbo.visit`.
+    
+    > You can also use destructuring to import only the functions you need. In
+    > this case, this would be: `import { visit } from '@hotwired/turbo'`.
```

```suggestion
    > this case, that would be: `import { visit } from '@hotwired/turbo'`.
```

(avoiding two `this` words too close together :) )